lib/taskana-core/src/test/java/acceptance/task/TransferTaskAccTest.java

```diff
@@ -271,21 +277,23 @@ public void testBulkTransferTaskWithExceptions()
     }
 
     @WithAccessId(userName = "teamlead_1")
-    @Test(expected = NotAuthorizedException.class)
-    public void testBulkTransferTaskWithoutAppendPermissionOnTarget()
-        throws InvalidArgumentException, WorkbasketNotFoundException, NotAuthorizedException {
+    @Test
+    public void testBulkTransferTaskWithoutAppendPermissionOnTarget() {
         TaskService taskService = taskanaEngine.getTaskService();
         ArrayList<String> taskIdList = new ArrayList<>();
         taskIdList.add("TKI:000000000000000000000000000000000006"); // working
         taskIdList.add("TKI:000000000000000000000000000000000041"); // NotAuthorized READ
 
-        try {
-            taskService
-                .transferTasks("WBI:100000000000000000000000000000000010", taskIdList);
-        } catch (NotAuthorizedException e) {
-            assertTrue(e.getMessage().contains("APPEND"));
-            throw e;
-        }
+        Assertions.assertThrows(NotAuthorizedException.class, () -> {
```

Comment:
This could be done better with a second assertion if the the exception is stored in a variable.

Reply:
Excellent Catch ... Done